packages/npm-dependencies-validation/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@sap-devx/npm-dependencies-validation",
+  "version": "1.3.0",
+  "private": true,
+  "description": "",
+  "author": "SAP SE",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/app-studio-toolkit"
```

Comment:
is there a way to link to a sub-folder in a repo

Reply:
Done